### PR TITLE
Added first batch of QC terms for mzQC in PSI-MS-CV

### PR DESCRIPTION
**Pull request by mwalzer**
This is the first batch of QC related terms in use with mzQC. As discussed in #80 we'd like to integrate into PSI-MS-CV and changed our mzQC terms as much as possible to align with the way terms are represented in PSI-MS-CV.
The first batch of terms is for one a rough cross-section of how we designed terms and concepts and also the extent to which we integrated QuaMeter metrics. 
As also discussed in #80 and the subsequent telco, we added the terms in a separate section of the namespace-id `MS:$sequence(7,4000000,4999999)$`. We then pinned our terms to the PSI-MS-CV root as a side-branch with:
```
[Term]
id: MS:4000000
name: PSI-MS CV Quality Control Vocabulary
def: "PSI Quality Control controlled vocabulary term." [PSI:QC]
relationship: part_of MS:0000000 ! Proteomics Standards Initiative Mass Spectrometry Vocabularies
```

Other notable introductions:
We added Typedef `has_value_concept`  where `has value type` is not enough; e.g.: where a float value represents a ratio. 
We added Typedefs `has_column` and `has_optional_column` to reflect the table value type as explained in #80 and the telco.
We introduced imports for the units and concepts, namely unit ontology, the statistical methods ontology, and the NCI thesaurus ontology.

You can find examples of how we use the terms in mzQC for example [here](https://github.com/HUPO-PSI/mzQC/blob/main/doc/examples/QC2-sample-example.mzQC).

### CODE REVIEW

**Comment by edeutsch** [comment]
For public visibility, we discussed the proposed terms last Friday and came up with some suggestions on how to improve them. Here is the sheet we worked through: https://docs.google.com/spreadsheets/d/15v694KU_1Fx7cmCEtgynIaqUpK13feCIA0xaemZoNF4/edit#gid=0
Wout was going to discuss these with the mzQC team to see if they can be accommodated.

**Comment by mwalzer** [comment]
Thank you @edeutsch and the PSI-MS-CV group for the swift consideration of the PR!
We discussed the suggestions and incorporated all but one, *not* changing the single value type name to scalar.
We hope the changes are sufficient.

File: psi-ms.obo [hunk 1]
```diff
@@ -20,12 +21,20 @@ remark: creator: Fredrik Levander <fredrik.levander <-at-> immun.lth.se>
...
+remark: creator: Nils Hoffmann < nils.hoffmann <-at-> cebitec.uni-bielefeld.de>
+remark: creator: Julian Uszkoreit <julian.uszkoreit <-at-> ruhr-uni-bochum.de>
+remark: creator: Mathias Walzer <walzer <-at-> ebi.ac.uk>
 remark: publisher: HUPO Proteomics Standards Initiative Mass Spectrometry Standards Working Group and HUPO Proteomics Standards Initiative Proteomics Informatics Working Group
 remark: When appropriate the definition and synonyms of a term are reported exactly as in the chapter 12 of IUPAC orange book. See http://www.iupac.org/projects/2003/2003-056-2-500.html and http://mass-spec.lsu.edu/msterms/index.php/Main_Page
 remark: For any queries contact psidev-ms-vocab@lists.sourceforge.net
 remark: URL: https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo
 remark: This work is licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) license.
 remark: To view a copy of this license, visit https://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View, California, 94041, USA.
+import: http://ontologies.berkeleybop.org/uo.obo
```
Comment (mobiusklein) [inline psi-ms.obo:35]: This line is redundant with line 7.

File: psi-ms.obo [hunk 2]
```diff
@@ -59,6 +68,18 @@ id: has_structured_representation_in_format
...
+name: has_value_concept
+
+[Typedef]
+id: has_column
+name: has_column
+
+[Typedef]
+id: has_optional_column
+name: has_optional_column
+
```
Comment (mobiusklein) [inline psi-ms.obo:82]: The typedef for `has_metric_category` is missing

**Review by mobiusklein — CHANGES_REQUESTED** [review]
Beyond the two notes, everything else looks good.

File: psi-ms.obo [hunk 3]
```diff
@@ -20,12 +21,20 @@ remark: creator: Fredrik Levander <fredrik.levander <-at-> immun.lth.se>
...
+remark: creator: Mathias Walzer <walzer <-at-> ebi.ac.uk>
 remark: publisher: HUPO Proteomics Standards Initiative Mass Spectrometry Standards Working Group and HUPO Proteomics Standards Initiative Proteomics Informatics Working Group
 remark: When appropriate the definition and synonyms of a term are reported exactly as in the chapter 12 of IUPAC orange book. See http://www.iupac.org/projects/2003/2003-056-2-500.html and http://mass-spec.lsu.edu/msterms/index.php/Main_Page
 remark: For any queries contact psidev-ms-vocab@lists.sourceforge.net
 remark: URL: https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo
 remark: This work is licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) license.
 remark: To view a copy of this license, visit https://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View, California, 94041, USA.
+import: http://ontologies.berkeleybop.org/uo.obo
+import: http://purl.obolibrary.org/obo/stato.owl
+import: http://purl.obolibrary.org/obo/ncit.owl
```
Comment (edeutsch) [inline psi-ms.obo:37]: should we reference the obos here in the obo?
And then transform to owl only in the transformed owl?

File: psi-ms.obo (hunk 1 above)
Comment (mwalzer) [reply psi-ms.obo:35]: indeed, thanks. Should we move line36-37 after line7?

File: psi-ms.obo (hunk 2 above)
Comment (mwalzer) [reply psi-ms.obo:86]: added in b7f9ca66a11fd8bbe779e4ec665d538529a84873

File: psi-ms.obo (hunk 3 above)
Comment (mwalzer) [reply psi-ms.obo:37]: OBOfoundry, OLS, stato github, etc. only offer the owl version. 
NCIT is here (http://purl.obolibrary.org/obo/ncit.obo). However, not directly on the page with the purl link but on some other part of obofoundry it states for NCIT: `NCIt OBO Edition releases should be considered experimental.`
Unsure how to proceed.

Comment (edeutsch) [reply psi-ms.obo:37]: okay, that's fine, we'll leave as is.

**Review by edeutsch — APPROVED** [review]

File: psi-ms.obo (hunk 1 above)
Comment (mobiusklein) [reply psi-ms.obo:35]: For simplicity, that seems reasonable.

**Review by mobiusklein — CHANGES_REQUESTED** [review]

**Review by mobiusklein — APPROVED** [review]

**Pull request merged**